src/EnergyPlus/Furnaces.cc

```diff
@@ -9935,6 +9950,74 @@ namespace Furnaces {
 
 	}
 
+	void
+	SetMinOATCompressor(
+		int const FurnaceNum, // index to furnace
+		std::string const FurnaceName, // name of furnace
+		std::string const cCurrentModuleObject, // type of furnace
+		std::string const CoolingCoilType, // type of cooling coil
+		std::string const CoolingCoilName, // name of cooling coil
+		std::string const HeatingCoilType, // type of heating coil
+		std::string const HeatingCoilName, // name of heating coil
```

Comment:
Coils might not have been called yet when reading input for furnace. Could rewrite code to use coil index instead. Too much extra work.